frontend/public/locales/en/public.json

```diff
@@ -1128,6 +1128,12 @@
   "Used": "Used",
   "StorageClass": "StorageClass",
   "Total": "Total",
+  "VolumeAttributesClass modification failed": "VolumeAttributesClass modification failed",
+  "Failed to modify VolumeAttributesClass for this PersistentVolumeClaim.": "Failed to modify VolumeAttributesClass for this PersistentVolumeClaim.",
+  "VolumeAttributesClass application pending": "VolumeAttributesClass application pending",
+  "VolumeAttributesClass modification in progress": "VolumeAttributesClass modification in progress",
+  "VolumeAttributesClass \"{{target}}\" is pending application.": "VolumeAttributesClass \"{{target}}\" is pending application.",
```

Comment:
For the application pending alert:
"VolumeAttributesClass application in progress
 {{target}} is being applied to replace {{current}}. Your volume remains available during this update." WYDT @kevinhatchoua @cajieh ?